ner4opt/ner4opt.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pathlib
+from typing import List
+
+import joblib
+
+from .constants import Constants
+from .exceptions import InvalidModelError
+from .features import Featurizer
+from .utils import format_entities, generate_feature_dictionary, l2_augment_sentence, load_torch_model
+
+
+class Ner4Opt(object):
+    """Extract named entities for a given optimization problem.
+
+    The main objective of the Ner4Opt library is to generate named entities for optimization problems. There are six
```

Comment:
Move "There are .. to the next line.. "

Same comment throughout this Text Blob, split sentences/paragraphs over rows.